docs/unraid-os/troubleshooting/licensing-faq.md

```
@@ -0,0 +1,284 @@
+</details><br />
+
+---
+
+## Trial Licenses
+
+<a id="trial-license" class="anchor-fix" style={{ fontWeight: "bold" }}><h3>How do Unraid trials work?</h3></a>
+
+<details>
+<summary>Click to expand/collapse</summary>
+
+- Trial licenses last 30 days and provide full Unraid Pro functionality without a storage device limit. 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)